aws_lambda_powertools/metrics/functions.py

```diff
@@ -69,3 +72,63 @@ def extract_cloudwatch_metric_unit_value(metric_units: List, metric_valid_option
         unit = unit.value
 
     return unit
+
+
+def validate_emf_timestamp(timestamp: int | datetime) -> bool:
```

Comment:
Is this validation time zone aware? Do we know which time zone CloudWatch uses to decide whether to drop or keep the metric?

Reply:
Thanks for catching that specific and critical detail, @dreamorosi! We should definitely account for timezones. I've updated the code accordingly